buildSrc/src/main/groovy/io/micronaut/guides/GuideAsciidocGenerator.groovy

```diff
@@ -236,10 +239,11 @@ class GuideAsciidocGenerator {
         features[0]
     }
 
-    private static List<String> featuresForApp(GuideMetadata metadata,
+    private static List<String> featuresForApp(Guide metadata,
                                                GuidesOption guidesOption,
                                                String app) {
-        List<String> features = metadata.apps.find { it.name == app }?.getVisibleFeatures(guidesOption.language) ?: []
+        App appEl = metadata.apps().find { it.name() == app }
+        List<String> features = appEl?GuideUtils.getAppVisibleFeatures(appEl,guidesOption.language):[]
```

Comment:
```suggestion
        List<String> features = appEl ? GuideUtils.getAppVisibleFeatures(appEl, guidesOption.language) : []
```
Please give spaces so that it is easy to read.